docs/kubernetes_deployment.md

```diff
@@ -1,6 +1,7 @@
+**IMPORTANT** This guide is deprecated and not updated any more. We strongly recommend using [Harbor Helm Chart](https://github.com/vmware/harbor/tree/master/contrib/helm/harbor) to deploy latest Harbor release on Kubernetes.
 
 ## Integration with Kubernetes
-This Document decribes how to deploy Harbor on Kubernetes.  It has been verified on **Kubernetes v1.6.5** and **Harbor v1.2.0**
+This Document decribes how to deploy Harbor on Kubernetes. It has been verified on **Kubernetes v1.6.5** and **Harbor v1.2.0**
```

Comment:
Could you please also update to point out it's the offline installer of harbor v1.2.0